.github/actions/oblt-cli-create-serverless/action.yml

```diff
@@ -37,71 +37,50 @@ inputs:
 runs:
   using: "composite"
   steps:
-
     - name: Configure oblt-cli flags
-      run: |
-        {
-          [ -n '${{ inputs.cluster-name-prefix }}' ] \
-            && echo -n '--cluster-name-prefix=${{ inputs.cluster-name-prefix }} '
-          [ -n '${{ inputs.cluster-name-suffix }}' ] \
-            && echo -n '--cluster-name-suffix=${{ inputs.cluster-name-suffix }} '
-          [ '${{ inputs.dry-run }}' == 'true' ] \
-            && echo -n '--dry-run '
-        } > .flags
-        echo "COMMAND=$(cat .flags)" >> $GITHUB_ENV
-        rm .flags
-      shell: bash
-
-    - name: Configure oblt-cli flags for gitops - issues
-      if: ${{ !github.event.issue.pull_request && contains(inputs.gitops, 'true') }}
-      run: |
-        {
-          echo -n '{'
-          echo -n '"GitOps":"true",'
-          echo -n '"GitHubRepository":"${{ github.repository }}"'
-          [ -n '${{ github.event.issue.number }}' ] && echo -n ',"GitHubIssue":"${{ github.event.issue.number }}"'
-          echo -n '}'
-        } > .gitops
-        echo "GITOPS=$(cat .gitops)" >> $GITHUB_ENV
-        rm .gitops
-      shell: bash
-
-    - name: Configure oblt-cli flags for gitops - pull-request
-      if: github.event.issue.pull_request && contains(inputs.gitops, 'true')
-      run: |
-        {
-          echo -n '{'
-          echo -n '"GitOps":"true",'
-          echo -n '"GitHubRepository":"${{ github.repository }}",'
-          echo -n '"GitHubCommit":"${{ github.event.comment.id }}",'
```

Comment:
This is not supported, I could not find where we use it so far. And I could not find how to gather the sha commit. Somehow I feel we don't use it and we don't need it anyway